src/test/java/uk/gov/hmcts/reform/sandl/snlevents/common/OurMockMvc.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.hmcts.reform.sandl.snlevents.common;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Component;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+//consider better name for class
```

Comment:
if for 5 days you have not come with better name then remove the coomment and leave this as is. it looks then like the best name we could come with...

Reply:
MockMvcEventsClient I propose